File: src/main/java/com/partymakers/shareparty/application/party/dto/PartyFriends.java
package com.partymakers.shareparty.application.party.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.partymakers.shareparty.domain.friends.entity.Friend;

import java.util.Set;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PartyFriends {
    @JsonProperty("friends")
    private Set<Friend> friends;
}

File: src/main/kotlin/com/partymakers/shareparty/party/presentation/dto/PartyFriends.kt
package com.partymakers.shareparty.party.presentation.dto

import com.fasterxml.jackson.annotation.JsonProperty
import com.partymakers.shareparty.friends.domain.entity.Friend

internal data class PartyFriends(
    @JsonProperty("friends")
    val friends: Set<Friend>
) 